app/scripts/modules/amazon/src/serverGroup/aws.less

```diff
@@ -0,0 +1,39 @@
+@import (reference) '~core/presentation/less/imports/commonImports.less';
+
+.collapsibleSubsectionOverride {
```

Comment:
Please scope this css using the name of the component that uses it, i.e.:
```
.InstancesDiversificationDetailsSection {
  .collapsibleSubsectionOverride {
   ... etc
```

Reply:
The plan is to use `collapsibleSubsectionOverride` in more components in upcoming feature additions. That's the reason I kept it generic. 
In the current PR, it is used for multiple instance types. In near future, it will be used for fields with multiple properties - this will happen with [multiple launch templates](https://docs.aws.amazon.com/autoscaling/ec2/userguide/asg-launch-template-overrides.html). 

Do you still think it should be scoped?

Reply:
> The plan is to use collapsibleSubsectionOverride in more components in upcoming feature additions.

I see, it makes sense why you write it that way.  I think my preference would be to encapsulate the CSS into a component (like `MyAwsCollapsibleSection` but with a name that explains what it does or why it exists).  If for some reason that is not possible, just rename the CSS class  to something more explicit (a top-level CSS class named `collapsibleSubsectionOverride` doesn't tell me much).

Reply:
Gotcha. I will give that a shot. Thanks.

Reply:
@christopherthielen renamed to `aws-multiple-values-subsection` - looks better now. Thanks.